javascript/builtins/Reflect.json

```diff
@@ -172,6 +184,9 @@
           "__compat": {
             "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Reflect/deleteProperty",
             "spec_url": "https://tc39.es/ecma262/multipage/reflection.html#sec-reflect.deleteproperty",
+            "tags": [
+              "web-features:snapshot:ecmascript-2015"
```

Comment:
Seems right, it's in https://262.ecma-international.org/6.0/#sec-reflect.deleteproperty but nowhere in https://262.ecma-international.org/5.1/.